packages/mobile/src/config/app-linking/config.ts

```diff
@@ -0,0 +1,29 @@
+import { RootScreenName, MainScreenName } from '~/common/enums/enums';
+
+const config = {
+  screens: {
+    [RootScreenName.MAIN_WITH_MENU]: {
+      screens: {
+        [RootScreenName.MAIN]: {
+          screens: {
+            [MainScreenName.HOME]: 'home',
+            [MainScreenName.ACCOUNT_ROOT]: {
+              screens: {
+                [RootScreenName.SIGN_IN]: 'sign-in',
+                [RootScreenName.SIGN_UP]: 'sign-up',
+              },
+            },
+          },
+        },
+      },
+    },
+    [RootScreenName.ITEM_INFO]: {
+      path: 'items/:itemId?',
+      parse: {
+        itemId: (itemId: string) => itemId,
+      },
```

Comment:
Params are parsed as strings **by default**

Reply:
![ts_error](https://user-images.githubusercontent.com/69375560/189832755-c4e5ad09-8c1a-4148-a3d5-5fabd981d7c0.jpg)
I get error from TS if I don't specify  it